bin/ytdl.js

```diff
@@ -296,6 +297,14 @@ if (opts.infoJson) {
       if (!ext && format.container) {
         output += '.' + format.container;
       }
+
+      // Parses & sanitises output filename for any illegal charcters
+      var parsedOutput = path.posix.parse(output);
```

Comment:
Why parse with `posix`? From the description you linked, it looks like this wouldn't parse correctly on Windows with a Windows styled path

Reply:
Nice catch. I blindly assumed that statement applied to the package as a whole. `posix` shouldn't be used for parse.

![conemu64_2017-11-29_12-42-13](https://user-images.githubusercontent.com/16233822/33353655-ea134dbc-d502-11e7-9619-57e9b3d11cd0.png)